account-kit/rn-signer/src/utils/mmkv-localstorage-polyfill.ts

```diff
@@ -0,0 +1,41 @@
+import { MMKV } from "react-native-mmkv";
+
+class MMKVLocalStorage {
+  private storage;
+  constructor() {
+    this.storage = new MMKV({ id: "mmkv-localstorage" });
+  }
+
+  // implement the Storage interface
+  getItem(key: string) {
+    return this.storage.getString(key) ?? null;
+  }
+
+  setItem(key: string, value: string) {
+    return this.storage.set(key, value);
+  }
+
+  removeItem(key: string) {
+    if (this.storage.contains(key)) {
+      this.storage.delete(key);
+    }
+  }
+
+  clear() {
+    this.storage.clearAll();
+  }
+
+  get length() {
+    return this.storage.getAllKeys().length;
+  }
+
+  key(index: number) {
+    const keys = this.storage.getAllKeys();
```

Comment:
I'm guessing `getAllKeys` returns a key value pair array?

Reply:
It returns an array of just the available keys, without their values. Here's from their docs: https://github.com/mrousavy/react-native-mmkv?tab=readme-ov-file#keys